test/e2e/secret_test.go

```diff
@@ -36,6 +36,21 @@ var _ = Describe("Podman secret", func() {
 		inspect.WaitWithDefaultTimeout()
 		Expect(inspect).Should(Exit(0))
 		Expect(inspect.OutputToString()).To(ContainSubstring("opt1:val"))
+
+		session = podmanTest.Podman([]string{"secret", "create", "-d", "file", "--driver-opts", "opt1=val1", "a", secretFilePath})
```

Comment:
Can we also test that labels are updated?  Create the initial one with `label foo` and the updated one with `label bar`.

We should also document whether the labels will be entirely replaced or added.